scrunch/datasets.py

```diff
@@ -1573,12 +1573,13 @@ def create_categorical(self, categories, alias, name, multiple, description='',
                 sv['case'] = get_else_case(sv['case'], categories)
 
                 if 'missing_case' in sv:
-                    not_selected_case = 'not ({}) and not ({})'.format(sv['case'], sv['missing_case'])
+                    selected_case = '({}) and not ({})'.format(sv['case'], sv['missing_case'])
+                    not_selected_case = 'not ({})'.format(selected_case)
```

Comment:
Small error in my suggested fix. We need to remove the parentheses around the insertion of `selected_case`.

Instead of this:

`not_selected_case = 'not ({})'.format(selected_case)`

It should be this:

`not_selected_case = 'not {}'.format(selected_case)`

Reply:
oh I see. This was my mistake I added the parentheses thinking we wanted to wrap it all. removing and updating tests now.